compiler/AST/symbol.cpp

```diff
@@ -1694,6 +1695,7 @@ void FnSymbol::finalizeCopy(void) {
       /*
        * Iterate over the statements that have been added to the function body
        * and add them to the new body.
+       * (Added only if needTupleInBody - see handleSymExprInExpandVarArgs().)
```

Comment:
This extension to the comment doesn't really add anything unless one has a lot of
understanding about var-arg expansion.  Please either remove it or expand it.